.github/workflows/enterprise.yml

```diff
@@ -0,0 +1,64 @@
+name: Enterprise build
+on:
+  push:
+    branches: [develop]
+
+jobs:
+  enterprise:
+    name: Enterprise build
+    runs-on: [ubuntu-latest]
+    env:
+      # REQUIRED, AppCenter application name, eg. "futured/Gastromapa"
+      APPCENTER_APP_NAME: futured/app-name
+      # Add additional AppCenter user groups separated by semicolon, if applicable, eg. "Collaborators;Partners"
+      APPCENTER_GROUPS: Collaborators
+      EXCLUDE_APK_FILTER: .*unaligned.apk\|.*Test.*.apk
+      # REQUIRED, Platform-specific slack channel name for notifications, eg. "gmlh-android"
+      SLACK_CHANNEL: project-slack-channel-name
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.6'
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Generate build number
+        uses: einaregilsson/build-number@v2
+        with:
+          token: ${{secrets.github_token}}
+      - name: Run check and Assemble APK
+        env:
+          BITRISE_BUILD_NUMBER: ${{ env.BUILD_NUMBER }}
+        shell: bash
+        run: ./gradlew lintCheck assembleDevEnterprise
+      - name: Run unit tests
+        shell: bash
+        run: ./gradlew --continue testDevEnterpriseUnitTest
+      - name: Find artifacts and mapping file
+        shell: bash
+        run: |
+          echo ::set-output name=apk_file::$(find . -name "*.apk" | grep -v ${{env.EXCLUDE_APK_FILTER}})
+          echo ::set-output name=aab_file::$(find . -name "*.aab")
+          echo ::set-output name=mapping_file::$(find . -name mapping.txt)
+        id: artifacts
+      - name: Publish to App Center
+        uses: okalman/AppCenter-Github-Action@master
+        with:
+          appName: ${{env.APPCENTER_APP_NAME}}
+          token: ${{secrets.APP_CENTER_TOKEN}}
```

Comment:
I think we should add a comment about secrets which are required to run these workflows because it's kinda hidden in code.